src/contexts/networkContext.ts

```diff
@@ -1,13 +1,12 @@
 import { Web3Provider } from '@ethersproject/providers'
 
-import { NetworkName } from 'models/network-name'
 import { createContext } from 'react'
 
 export const NetworkContext: React.Context<{
   signingProvider?: Web3Provider
-  signerNetwork?: NetworkName
   userAddress?: string
```

Comment:
i'd _really_ like to change `userAddress` to `connectedWallet`. haven't done so to avoid scope creep, but can we plz?

Reply:
Suggestion: Could do a stacked PR, on top of this PR that just changes the name of `userAddress` to `connectedWallet` so it's in a different scope and doesn't get mixed in with this feature work.

Also, not trying to push anything on anyone, I will stop making the suggestion after dis.